src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.js

```diff
@@ -38,38 +38,50 @@ const VideoSpotlightBlock = ({
   button,
   image,
   playButtonLink,
-}) => (
-  <div className={s.bleedWrapper}>
-    <div className={s.bleedImage}>
-      <div className={s.playButton}>
-        <a rel="noreferrer" href={playButtonLink} target="_blank">
-          <img className={s.playIcon} src={playIcon} alt="Play Video" />
-        </a>
+  blackBackground,
+}) => {
+  const bleedBackgroundWrapper = `${s.bleedWrapper} ${
+    blackBackground ? s.blackBackground : ""
+  }`;
```

Comment:
Just an observation here, and no need to change anything here, but maybe it's about time we look at using the [`classnames`](https://github.com/JedWatson/classnames) library, which makes this conditional class name computation a bit cleaner.

See https://github.com/JedWatson/classnames/tree/b166fd132d4a46b6ed93b3264ca638c7f14d4620#usage-with-reactjs for an example of how this can be used, but it works pretty well with React.

I can create a ticket to investigate this later.

Reply:
ok